web/migrations/0059_question_choice_response_survey_question_survey.py

```
@@ -0,0 +1,83 @@
+            ],
+        ),
+        migrations.CreateModel(
+            name="Survey",
+            fields=[
+                ("id", models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("title", models.CharField(max_length=200)),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                (
+                    "author",
+                    models.ForeignKey(
+                        blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)